api/app/controllers/api/v1/assignments_controller.rb

```diff
@@ -16,6 +16,11 @@ def index
 
         # POST /assignments
         def create
+            # if we passed in an id that exists, we want to update
+            if params[:id] && Assignment.exists?(params[:id])
```

Comment:
Again, if an invalid id is passed in, won't this just do a regular create?

Reply:
Yes. I'm assuming that what we want(?) The initial structure for this setup was made by @siefkenj, so you have to ask him about the specifics on why it was designed this way.

Reply:
Hmm, this will have to be tested and kept consistent on live. If this is the supposition, we need to make sure that this is confirmed.

Reply:
I got rid of the `create` rerouting to `update` and made the routes directly call `POST /tables/:id` instead.